src/components/form/FormBase.tsx

```diff
@@ -44,10 +44,6 @@ export function FormBase<TInitialValues extends MaybeModelWithAttributes>({
         }
     }, [customAttributes.data, initialValues, includeAttributes])
 
-    if (customAttributes.error) {
```

Comment:
Why is this removed?

Reply:
Ah, I think I understand now - indicator type dont have attributes. Should be enough to pass `includeAttributes={false}` to the BaseForm component.